src/app.py

```diff
@@ -76,6 +77,14 @@ def __index():
         def __search():
             return self.search()
 
+        @self.app.route('/mapping', methods=['GET'])
+        def __mapping():
+            return self.mapping()
+
+        @self.app.route('/mapping/<index_name>', methods=['GET'])
```

Comment:
To be consistent with other endpoints, we should use `/<index_without_prefix>/mapping`.